docs/current/guides/324301-argo-workflows.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: /324301/argo-workflows
+displayed_sidebar: "current"
+category: "guides"
+tags: ["argo", "kubernetes"]
+authors: ["Kyle Penfound"]
+date: "2023-08-22"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Use Dagger with Argo Workflows
+
+## Introduction
+
+[Argo Workflows](https://argoproj.github.io/argo-workflows/) is an open source container-native workflow engine for orchestrating parallel jobs on Kubernetes. This guide explains how to run Dagger pipelines in Argo Workflows.
+
+## Requirements
+
+This guide assumes that you have a basic understanding of Kubernetes and Argo Workflows, and that your Kubernetes cluster has been configured following the [Run Dagger on Kubernetes](https://docs.dagger.io/194031/kubernetes) guide.
+
+## Step 1: Install Argo Workflows
+
+The first step is to install [Argo Workflows](https://argoproj.github.io/argo-workflows/) in the Kubernetes cluster.
+
+Follow the [Argo Workflows quickstart](https://github.com/argoproj/argo-workflows/blob/master/docs/quick-start.md) steps, adjusting them as needed to your own requirements. Once you've successfully installed Argo Workflows in your cluster, continue to the next step.
+
+## Step 2: Run a sample workflow
+
+The sample workflow will clone and run the CI for the [greetings-api](https://github.com/kpenfound/greetings-api) demo project. This project uses the Dagger Go SDK for CI.
+
+Create a file called `workflow.yaml` with the following content:
+
+```yaml file=./snippets/argo-workflows/workflow.yaml
+```
+
+A few important points to note:
+
+- The workflow uses hardwired artifacts to clone the Git repository and to install the Dagger CLI.
+- `unix:///var/run/dagger/buildkitd.sock` is mounted and specified with the `_EXPERIMENTAL_DAGGER_RUNNER_HOST` environment variable.
+- The Dagger CLI `dagger_v0.8.7_linux_amd64.tar.gz` is downloaded and installed. Confirm the version and architecture are accurate for your cluster and project.
+- The image `golang:1.21.0-bookworm` is used as the runtime for the container because the example project requires Go.
+
+The workflow uses a PersistentVolumeClaim for the runtime dependencies of the pipeline. Even though the dependencies within the pipeline are cached through Dagger, we still have dependencies to run the pipeline.
```

Comment:
```suggestion
The workflow uses a PersistentVolumeClaim for the runtime dependencies of the pipeline.
```

Reply:
Why?

Reply:
It reads clearer to me without the second sentence, but I concede this will be subjective.

Reply:
See if my edit is better?

Reply:
Pushed a change!